crates/matrix-sdk/src/send_queue/upload.rs

```diff
@@ -266,18 +261,20 @@ impl QueueStorage {
             if let Some((info, new_source)) =
                 thumbnail_info.as_ref().zip(sent_media.thumbnail.clone())
             {
-                let from_req =
-                    Media::make_local_thumbnail_media_request(&info.txn, info.height, info.width);
+                // Previously the media request used `MediaFormat::Thumbnail`. Handle this case
+                // for send queue requests that were in the state store before the change.
```

Comment:
state store? or cache store?

Reply:
This is about `DependentQueuedRequestKind`, so the state store, right?

Reply:
Ah, you're right :facepalm: